src/EnergyPlus/GroundTemperatureModeling/GroundTemperatureModelManager.hh

```diff
@@ -61,6 +61,8 @@
 
 namespace EnergyPlus {
 
+    struct DataGlobal;
```

Comment:
I had to go around a number of places and add this forward declaration.  It had previously worked because these files all included `DataGlobals.hh`, which then included the main data file.  When I took that out, these did not compile anymore.  This new pattern works fine though.